packages/engine/src/interaction/systems/InteractableSystem.ts

```diff
@@ -232,12 +190,12 @@ const execute = () => {
 
 export const InteractableSystem = defineSystem({
   uuid: 'ee.engine.InteractableSystem',
-  insert: { before: TransformSystem },
+  insert: { after: TransformSystem },
```

Comment:
why has this been moved to after the transform system?

Reply:
reverted due to fixing the orbitcam issue